components/molecules/InsightHeader/insight-header.tsx

```diff
@@ -22,9 +23,15 @@ const InsightHeader = ({ insight, repositories }: InsightHeaderProps): JSX.Eleme
         <ContextThumbnail size={120} ContextThumbnailURL={""}></ContextThumbnail>
       </div>
       <div className="header-info flex flex-col grow justify-center p-2">
-        <Title level={1} className="!text-3xl font-semibold tracking-tight text-slate-900">
-          {insight && insight.name || "Insights"}
-        </Title>
+        <div className="flex items-baseline">
```

Comment:
```suggestion
        <div className="flex items-baseline gap-4">
```

Reply:
Using the `gap-4` class would handle the space between the Page Name and the Label, but not really sure 4 would match though... you can try out other sizes :) 😇

Reply:
this was very helpful thanks, gap 4 was just the right property to use.